scripts/install-test-template.sh

```diff
@@ -34,19 +34,24 @@ done
 # Use tarballs to ensure that published packages are consumable
 npm pack
 
-yarn ci
+yarn
 yarn react-native init-test-app --destination template-example --name TemplateExample --platform $platform
-cp .yarnrc-offline template-example/.yarnrc
 
 pushd template-example 1> /dev/null
 
+echo 'yarnPath: "../.yarn/releases/yarn-berry.cjs"' >> .yarnrc.yml
```

Comment:
what's this yarn-berry file btw? isn't berry yarn 2?

Reply:
It's apparently the "current" version:
```sh
% .yarn/releases/yarn-berry.cjs --version
3.0.0
```
With Yarn 3.0, it's recommended to check in a local copy. Your Yarn installation will automatically pick it up.

Reply:
The `yarn-berry.cjs` is the name `yarn` uses when you first run `yarn set version latest` from `v1`. If you were to run it again from a `v2` or `v3` it'll switch over to using a versioned file name.

![image](https://user-images.githubusercontent.com/208549/129276292-44749eec-49cb-4c98-b03f-fa585e25723c.png)